libbeat/processors/kubernetes/_meta/fields.yml

```diff
@@ -0,0 +1,29 @@
+- key: kubernetes
+  title: Kubernetes info
+  description: >
+    Kubernetes metadata added by the kubernetes processor
+  fields:
+    - name: kubernetes.pod.name
+      type: keyword
+      description: >
+        Kubernetes pod name
+
+    - name: kubernetes.namespace
+      type: keyword
+      description: >
+        Kubernetes namespace
+
+    - name: kubernetes.labels
+      type: object
+      description: >
+        Kubernetes labels map
+
+    - name: kubernetes.annotations
```

Comment:
I've been reading https://www.elastic.co/guide/en/beats/libbeat/5.3/event-conventions.html and also saw how we do it in docker module: https://github.com/elastic/beats/blob/master/metricbeat/module/docker/container/_meta/data.json#L13

It sounds more natural to me to have labels as plural. It's like you are naturally selecting a label by name from the labels dict, what do you think?